### PR TITLE
Frames GS: Improve TSDB R/W examples & doc + no delete-by-filter [IG-12272 IG-13893 IG-12092]

### PR DESCRIPTION
**Pull request by Sharon-iguazio**
**getting-started/frames.ipynb** — I replaced the TSDB R/W examples and made related doc edits.
(The diff also shows changes in the NoSQL example outputs because I re-ran all code cells and the data 
@adiso75 @talIguaz please approve.

### CODE REVIEW

**Comment by adiso75** [comment]
@Sharon-iguazio  in section 16 you have two read examples:
why do you need this one? where do you define q_start?
df = client.read(backend="tsdb", query=query_str, step="1h", start=q_start,
                 end="now", multi_index=True)

**Comment by Sharon-iguazio** [comment]
> @Sharon-iguazio in section 16 you have two read examples:
> why do you need this one? where do you define q_start?
> df = client.read(backend="tsdb", query=query_str, step="1h", start=q_start,
> end="now", multi_index=True)

@adiso75 the second read command (with `q_start`) was used for test purposes (along with a related variable definition) to ensure consistent results with `now` queries; it wasn't supposed to be committed. Good catch! I've now removed it (see commit 155ecd0).

**Comment by Sharon-iguazio** [comment]
NOTE: We've noticed that the format of the randomly generated `time` values in the currently implementation is not standard RTC, which is probably what messes up the ability to use `write` `index_cols` parameter to set the index columns. => Tal will look for a solution.

**Comment by Sharon-iguazio** [comment]
@adiso75 @talIguaz I made a slight change to the TSDB `write` example's data generation to remove the explicit setting of the time zone in order to bypass the issue with Bug IG-14253. I verified that I can write the times generated with the modified code using `write` with the `index_cols` parameter.

**Comment by Sharon-iguazio** [comment]
@adiso75  @talIguaz I also edited the `delete` bullet in the overview doc cell to remove the reference to deleting specific items ("delete by filter" not yet supported).

File: getting-started/frames.ipynb [hunk 1]
```diff
@@ -990,102 +991,88 @@
...
     "\n",
-    "For both types of queries, you can also optionally set additional parameters.\n",
-    "`start` and `end` define the query's time range &mdash; the metric-sample timestamps to which to apply the query (the default end time is `\"now\"` and the default start time is 1 hour before the end time); `step` defines the interval for aggregation or raw-data downsampling (default = the query's time range); and`aggregationWindow` defines the aggregation time frame for over-time aggregation (default = the aggregation step).<br>\n",
-    "You can set the optional `multi_index` parameter to `True` to return labels as index columns, as demonstrated in the following example.\n",
-    "By default, only the metric sample-time primary-key attribute is returned as an index column.<br>\n",
-    "See the [Frames documentation](https://github.com/v3io/frames/blob/master/README.md) for more information about the supported parameters of the `read` method for the `tsdb` backend."
+    "# Genearte a DataFrame with TSDB metric samples and a \"time\" index column\n",
+    "def gen_df_w_tsdb_data(num_items=24, freq=\"1H\", end=None, start=None,\n",
```
Comment (talIguaz) [inline getting-started/frames.ipynb]: if you created the table with `1/m` rate better create the data the same way (with 1m freq) for consistency.

File: getting-started/frames.ipynb [hunk 2]
```diff
@@ -1107,69 +1094,1600 @@
...
+       "2019-12-06 04:44:20.869 11   windows  83.516672  23.327160  93.248763"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Read the full table contents, as in the previous example but use an SQL query\n",
+    "query_str = os.path.join(\"select * from '\" + tsdb_table + \"'\")\n",
```
Comment (talIguaz) [inline getting-started/frames.ipynb]: `query_str = "select * from ' + tsdb_table + ')`

File: getting-started/frames.ipynb (hunk 1 above)
Comment (Sharon-iguazio) [reply getting-started/frames.ipynb]: @talIguaz as we agreed f2f, I changed the `rate` in the `create` example to `1h` to match the write data set.

File: getting-started/frames.ipynb (hunk 2 above)
Comment (Sharon-iguazio) [reply getting-started/frames.ipynb]: @talIguaz changed in commit b9378e8, which also makes other related cosmetics updates (`os.path.join` — receive path variables; use `f` for formatting with variables in more locations; reduce empty lines at end of cells). I did a full-execution test of all cells and everything worked well.

File: getting-started/frames.ipynb [hunk 3]
```diff
@@ -111,7 +111,7 @@
    "outputs": [],
    "source": [
     "# Relative path to the NoSQL table within the parent platform data container\n",
-    "table = os.path.join(os.getenv(\"V3IO_USERNAME\") + \"/examples/bank\")\n",
+    "table = os.path.join(os.getenv(\"V3IO_USERNAME\"), \"/examples/bank\")\n",
```
Comment (talIguaz) [inline getting-started/frames.ipynb]: you dont need the slash before the `examples/bank` same for the other place you changed

Comment (Sharon-iguazio) [reply getting-started/frames.ipynb]: @talIguaz missed 3 instances ... => FIXED (commit a409c9a).

**Comment by Sharon-iguazio** [comment]
Tal approved merging the PR.

**Pull request merged**